algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/model.py

```diff
@@ -0,0 +1,424 @@
+"""
+This is a pytorch implementation mirroring:
+https://github.com/google/init2winit/blob/master/init2winit/model_lib/conformer.py
+"""
+from dataclasses import dataclass
+import math
+from typing import Tuple
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from . import preprocessor
+from .spectrum_augmenter import SpecAug
```

Comment:
can we use absolute imports instead of relative? that way if we reorganize the dir structure in the future we don't break these

Reply:
Done.